### PR TITLE
Fix function name

### PR DESCRIPTION
**Pull request by ceth-x86**
Fixed issue #1. If you have any comments or remarks let me know.

### CODE REVIEW

File: healthchecker.go [hunk 1]
```diff
@@ -99,19 +99,17 @@ func toString(ts interface{}) string {
 	return s
 }
 
-func (h *Healthchecker) Add(healthfunc Healthfunc, nameFunction ...interface{}) {
-	nf := toString(nameFunction)
-
+func (h *Healthchecker) Add(healthfunc Healthfunc, nameFunction string) {
```
Comment (ervitis) [inline healthchecker.go]: If you don't mind, I would use an optional string parameter, like `nameFunction ...string` so I wouldn't broke so much the function

Comment (ervitis) [reply healthchecker.go]: As I have reviewed the code, the bad part (oh my gosh...) was that if it has a parameter it's inside the slice of the interface{} :man_facepalming:

**Comment by ceth-x86** [comment]
Hmm... I have an error, but I don't know the reason of it:

> The command "make check" exited with 2. 

I tried to run `golangci-lint` locally against `healthchecker.go` and didn't have any errors.

```pseudocode
/home/demas/go/bin/golangci-lint run -v healthchecker.go
INFO [config_reader] Config search paths: [./ /home/demas/development/github/my-gohealthchecker /home/demas/development/github /home/demas/development /home/demas /home /] 
INFO [config_reader] Used config file .golangci.yml 
INFO [lintersdb] Active 10 linters: [deadcode errcheck gosimple govet ineffassign staticcheck structcheck typecheck unused varcheck] 
INFO [lintersdb] Active 10 linters: [deadcode errcheck gosimple govet ineffassign staticcheck structcheck typecheck unused varcheck] 
INFO [loader] Go packages loading at mode 575 (types_sizes|files|imports|name|compiled_files|deps|exports_file) took 99.433547ms 
INFO [runner/filename_unadjuster] Pre-built 0 adjustments in 351.467µs 
INFO [runner/goanalysis_metalinter/goanalysis] analyzers took 0s with no stages 
INFO [runner/unused/goanalysis] analyzers took 0s with no stages 
INFO [runner] processing took 2.978µs with stages: max_same_issues: 857ns, nolint: 294ns, max_from_linter: 227ns, skip_dirs: 207ns, skip_files: 157ns, cgo: 156ns, path_shortener: 135ns, autogenerated_exclude: 128ns, identifier_marker: 126ns, path_prettifier: 123ns, source_code: 121ns, filename_unadjuster: 118ns, diff: 117ns, exclude-rules: 56ns, max_per_file_from_linter: 52ns, exclude: 52ns, uniq_by_line: 52ns 
INFO [runner] linters took 29.094313ms with stages: goanalysis_metalinter: 28.56666ms, unused: 421.557µs 
INFO File cache stats: 0 entries of total size 0B 
INFO Memory: 3 samples, avg is 69.7MB, max is 70.2MB 
INFO Execution took 132.881379ms
```

**Comment by ervitis** [comment]
Sometimes I get that error running with travis. I will set up with circle ci and see if it happens too.
I will fix it later

**Comment by ervitis** [comment]
Can we please talk about not using the optional parameter in the function `Add()`?

**Comment by ceth-x86** [comment]
Yes. I'll rewrite my code.

**Comment by ceth-x86** [comment]
I am not sure I understood correctly: 

> **not** using the optional parameter in the function Add()

In the previous code the second parameter was required. Now I changed the code and second parameter **is** optional.

If I got it wrong let me know.

**Comment by ervitis** [comment]
Sorry, my bad, what I meant is I would use an optional parameter, because the version v1.1.0 had it

**Comment by ceth-x86** [comment]
Ok, I have made second parameter as optional. In this version we don't need to check the type of this parameter, because Go will check it itself.

**Comment by ervitis** [comment]
Thank you so much

**Pull request merged**